tpch-setup.sh

```diff
@@ -94,12 +95,16 @@ do
 	i=`expr $i + 1`
 done
 STOPTIME2="`date +%s`"
-hive -i settings/load-${SCHEMA_TYPE}.sql -f ddl-tpch/bin_${SCHEMA_TYPE}/analyze.sql --database ${DATABASE};
+ANALYZE_COMMAND="hive -i settings/load-${SCHEMA_TYPE}.sql -f ddl-tpch/bin_${SCHEMA_TYPE}/analyze.sql --database ${DATABASE}";
+runcommand "$COMMAND"
 STOPTIME3="`date +%s`"
 
 echo "Data loaded into database ${DATABASE}."
 
 LOADTIMES_FILE=loadtimes.csv
 touch $LOADTIMES_FILE
 chmod 777 $LOADTIMES_FILE
-echo "${STARTTIME},${STOPTIME1},${STOPTIME2},${STOPTIME3}" >> $LOADTIMES_FILE;
+echo "STARTTIME,DBGEN,EXTERNALTABLELOAD,ORCLOAD,ANALYZETIME"
+echo "${STARTTIME1},${STARTTIME2},${STOPTIME1},${STOPTIME2},${STOPTIME3}" >> $LOADTIMES_FILE;
```

Comment:
May rename variable so that its more obvious what they represent? For example, STARTTIME2 --> DBGENSTARTTIME or something.